1.22.0 broke existing code type check

Existing code stopped working with 1.22.0, to be exact #553.

```typescript
bot.on(
	'edited_message',
	async ctx =>
		ctx.reply(
			'Edited!',
			{reply_to_message_id: ctx.editedMessage.message_id},
		),
);
```

```plaintext
error TS18048: 'ctx.editedMessage' is possibly 'undefined'.
```

This looks like a mistake and should be fixed.

--- a/src/filter.ts
+++ b/src/filter.ts
@@ -528,7 +528,7 @@ type FilteredContextCore<U extends Update> =
 // must be in sync with shortcut impl!
 interface Shortcuts<U extends Update> {
     message: [U["message"]] extends [object] ? U["message"] : undefined;
-    edited_message: [U["edited_message"]] extends [object] ? U["edited_message"]
+    editedMessage: [U["edited_message"]] extends [object] ? U["edited_message"]
         : undefined;
     channelPost: [U["channel_post"]] extends [object] ? U["channel_post"]
         : undefined;

